make/modules/java.desktop/gensrc/GensrcX11Wrappers.gmk

```diff
@@ -24,13 +24,13 @@
 #
 
 # Generate java sources using the X11 offsets that are precalculated in files
-# make/data/x11wrappergen/sizes-<address size>.txt.
+# src/java.desktop/share/data/x11wrappergen/sizes-<address size>.txt.
```

Comment:
This doesn't seem right to me. This is X11 specific. It should not be in share.
Same for related files.